config/clusters/pangeo-hubs/coessing.values.yaml

```diff
@@ -17,6 +17,26 @@ basehub:
             url: https://coessing.org/
             logo_url: "https://coessing.files.wordpress.com/2016/08/ghana-logo-21.png?w=262&h=376&zoom=2"
     singleuser:
+      initContainers:
+        # Need to explicitly fix ownership here, since EFS doesn't do anonuid
```

Comment:
This comment is referencing AWS EFS but this is actually living in GCP... I guess this is just a copy-pasted comment.